src/ServiceManagement/Compute/Commands.ServiceManagement/Automation/RunCodeGeneration.ps1

```diff
@@ -0,0 +1,459 @@
+﻿[CmdletBinding()]
+param(
+    [Parameter(Mandatory = $true)]
```

Comment:
Consider adding documentation attributes for your cmdlet - this will make help available for those using it